sdks/go/pkg/beam/core/funcx/output.go

```diff
@@ -84,8 +84,9 @@ func unfoldEmit(t reflect.Type) ([]reflect.Type, bool, error) {
 		if ok, err := isInParam(t.In(i)); !ok {
 			return nil, false, errors.Wrap(err, errIllegalParametersInEmit)
 		}
-		if t.In(i).Kind() == reflect.Interface || (t.In(i).Kind() == reflect.Pointer && t.In(i).Elem().Kind() == reflect.Interface) {
-			panic("Type interface{} isn't a supported PCollection type")
+		emptyInterface := reflect.TypeOf((*any)(nil)).Elem()
```

Comment:
feel free to declare this outside the function so it's only done once, rather than every time the method is called.

Reply:
Oh, got it! Thanks! I changed it.